code-studio/src/js/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -252,12 +257,13 @@ var WorkshopForm = React.createClass({
   },
 
   handleErrorClick: function (i) {
-    this.state.errors.splice(i,1);
-    this.setState(this.state);
+    var errors = _.cloneDeep(this.state.errors);
+    errors.splice(i,1);
+    this.setState({errors: errors});
```

Comment:
oh, you're right - i misunderstood what was happening (i thought you were trying to get that single error instead of remove that single error). ignore my suggestion